QandA/05-control_flow.Rmd

```diff
@@ -295,3 +294,107 @@ ifelse(c(TRUE, FALSE, TRUE, FALSE, TRUE, TRUE), 1:10, "FALSE")
 ```
 
 we can see that ifelse places the numbers in `1:10` based on their index where our condition is to `TRUE` and inserts the string `"FALSE"` whenever the condition is `FALSE`
+
+## Complexity {-}
+
+We can use the package `cyclocomp` to look at the cyclomatic complexity of our functions. Looking at each function from the Chapter 5 presentation:
```

Comment:
Possibly helpful to define cyclomatic complexity here as part of this chunk?

Reply:
Good idea - submitting something let me know what you think!